pkg/apis/openebs.io/v1alpha1/cas_pool.go

```diff
@@ -37,6 +37,11 @@ const (
 	StoragePoolClaimCPK CasPoolKey = "openebs.io/storage-pool-claim"
 	// CStorPoolClusterCPK is the CStorPoolcluster label
 	CStorPoolClusterCPK CasPoolKey = "openebs.io/cstor-pool-cluster"
+	// CStorPoolInstanceCPK is the CStorPoolInstance label
+	CStorPoolInstanceCPK CasPoolKey = "openebs.io/cstor-pool-instance"
```

Comment:
don't we need to set this label on pool pods? I don't see them being set

Reply:
ok.. we seem to be setting it.. can we change in getDeployLabels as well to macro?

Reply:
Updated with constant